regimes/ro/bill_invoice.go

```diff
@@ -0,0 +1,132 @@
+package ro
+
+import (
+	"github.com/invopop/gobl/bill"
+	"github.com/invopop/gobl/org"
+	"github.com/invopop/gobl/tax"
+	"github.com/invopop/validation"
+)
+
+// Romanian invoice validation based on:
+// - Law 227/2015 (Fiscal Code - Art. 319 Invoicing): https://legislatie.just.ro/Public/DetaliiDocument/171282
+// - Law 296/2023 (B2B e-Factura Mandate): https://legislatie.just.ro/Public/DetaliiDocumentAfis/275745
+// - OUG 69/2024 (B2C e-Factura Mandate effective Jan 1, 2025): https://legislatie.just.ro/Public/DetaliiDocument/283893
+// - OUG 138/2024 (Simplified Invoices Mandate effective Jan 1, 2025): https://legislatie.just.ro/Public/DetaliiDocument/293675
+// - OUG 120/2021 (e-Factura Framework & RO_CIUS): https://legislatie.just.ro/Public/DetaliiDocument/246848
+//
+// Key requirements:
+// - Supplier must have a valid Romanian tax ID (CIF/CUI).
+// - B2B Customers must have a valid tax ID (CUI).
+// - B2C Customers (individuals) are mandatory in e-Factura as of Jan 1, 2025 (OUG 69/2024).
+//   If a CNP is not provided, the invoice is still valid but reported with a standard placeholder (13 zeros).
+// - Simplified invoices (<100 EUR) are mandatory in e-Factura as of Jan 1, 2025 (OUG 138/2024).
+// - Reporting deadline is 5 calendar days from issuance.
+
+// validateBillInvoice validates Romanian invoices according to local requirements.
+func validateBillInvoice(inv *bill.Invoice) error {
```

Comment:
This looks very complete, but we don't normally add this level of validation at the Regime level and instead define an "addon" for a local format or tax agency.